sdk/core/Microsoft.Extensions.Azure/README.md

```diff
@@ -0,0 +1,54 @@
+# Azure client integration for ASP.NET Core
+
+Microsoft.Extensions.Azure.Core provides shared primitives to integrate Azure clients with ASP.NET Core dependency injection and configuration systems.
```

Comment:
Is there a good ASP.NET configuration doc on MSDN we could link folks too if they want to discover more?